public/app/exercise/exercise.controller.js

```diff
@@ -157,6 +159,13 @@
       exercise.editor.on('change', resetIdleLoop);
       resizeCodeMirror();
     };
+    
+    
+    function loadEditorContent() {
+      var editorValue = $window.localStorage.getItem("editor." + exercise.id + "." + editor.getOption("mode"));
+      if (editorValue != null) 
```

Comment:
It's one of the common best practices to use `===` and `!==` instead of `==` and `!=` to prevent type coercion during comparison.
For more details about type coercion and why we want to avoid it, you can check this link: http://blog.jeremymartin.name/2008/03/understanding-loose-typing-in.html